torch/_inductor/codegen/triton.py

```diff
@@ -2470,7 +2473,7 @@ def codegen_cooperative_reduction_peer_combine(self, result_var, dtype):
         column.  After the barrier, every thread block loads the completed value so that it can compute the final
         value independently.
         """
-        xnumel, rnumel = self.numels
+        xnumel, rnumel = self.numels.values()
```

Comment:
This should be updated to do dict access

Reply:
Updated [d3146d1](https://github.com/pytorch/pytorch/pull/141751/commits/d3146d175c2a6ebbeff582d1561c2539ea1cf91e)